chef/cookbooks/hyperv/recipes/windows_features.rb

```diff
@@ -15,6 +15,7 @@
   node[:features_list][:windows].each do |feature_list|
     windows_feature feature_list do
       action :install
+      all true
```

Comment:
@ovisan but that's not the cookbook we're using, we're using the `windows` cookbook included in this barclamp.

I guess we could sync the cookbook, though.